services/nuron-ai/src/nuron_ai/core.py

```
@@ -14,11 +14,13 @@
 AuthorSource = Literal["extracted", "default", "unknown"]
 DeltaAction = Literal["add", "update", "unchanged", "drop_ref"]
 
+# YAML frontmatter at file start: --- ... ---. Group 1 = block body.
 _FRONTMATTER_RE = re.compile(r"\A---[ \t]*\n(.*?\n)---[ \t]*\n?", re.DOTALL)
-# Dash class covers hyphen/en-dash/em-dash: markdown conversion (LlamaParse, A11) doesn't
-# preserve which one the source used.
+# Closing line: "- Name, YYYY-MM-DD". Dash class = hyphen/en/em; LlamaParse/A11 don't preserve which the source used.
 _SIGNATURE_RE = re.compile(r"^[-–—][ \t]*([^,\n]+),[ \t]*(\d{4}-\d{2}-\d{2})[ \t]*$", re.MULTILINE)
```

Ruff (0.16.3)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)